trade-finance-manager-ui/templates/case/tasks/task.njk

```diff
@@ -13,7 +13,7 @@
     <div class="govuk-grid-row" data-cy="deal-tasks">
 
       <h4 class="govuk-heading-s govuk-!-margin-bottom-2" data-cy="deal-heading">Deal</h4>
-      <p class="govuk-body" data-cy="deal-subheading"><a href="/case/{{ dealId }}/deal" data-cy="deal-subheading-link">{{ deal.details.ukefDealId }}</a> {{ deal.submissionDetails.supplierName }} > {{ deal.submissionDetails.buyerName }}</p>
+      <p class="govuk-body" data-cy="deal-subheading"><span class="govuk-visually-hidden">View deal</span><a href="/case/{{ dealId }}/deal" data-cy="deal-subheading-link">{{ deal.details.ukefDealId }}</a> {{ deal.submissionDetails.supplierName }} > {{ deal.submissionDetails.buyerName }}<span class="govuk-visually-hidden">details </span></p>
```

Comment:
At the moment new `View deal` `details` spans are outside link `a` element. To be consistent both new `spans` should be inside <a>.